### PR TITLE
Adjust locators to patternfly5

### PR DESCRIPTION
**Pull request by ebondare**
Fixes test_obc_creation_and_deletion[openshift-storage.noobaa.io-noobaa-default-bucket-class-Actions-True] and test_obc_creation_and_deletion[openshift-storage.noobaa.io-noobaa-default-bucket-class-three_dots-True] failing in 4.16

### CODE REVIEW

File: ocs_ci/ocs/ui/views.py [hunk 1]
```diff
@@ -628,7 +647,7 @@
         By.XPATH,
     ),
     "object_storage": ("//a[normalize-space()='Object Storage']", By.XPATH),
-    "observe": ("//button[text()='Observe']", By.XPATH),
+    "Monitoring": ("//button[text()='Monitoring']", By.XPATH),
```
Comment (DanielOsypenko) [inline ocs_ci/ocs/ui/views.py:650]: Monitoring need to be changed back to Observe; this is one of the recent changes probably accidentally pulled from the stale master branch

Please do rebase to master

File: ocs_ci/ocs/ui/views.py [hunk 2]
```diff
@@ -1807,7 +1843,6 @@
         "bucketclass": bucketclass,
         "topology": topology,
         "mcg_stores": mcg_stores,
-        "alerting": alerting,
```
Comment (DanielOsypenko) [inline ocs_ci/ocs/ui/views.py:1810]: same

File: ocs_ci/ocs/ui/views.py [hunk 3]
```diff
@@ -1855,7 +1890,6 @@
...
```
Comment (DanielOsypenko) [inline ocs_ci/ocs/ui/views.py:1858]: same

File: ocs_ci/ocs/ui/views.py (hunk 1 above)
Comment (ebondare) [reply ocs_ci/ocs/ui/views.py]: Sorry, fixed

File: ocs_ci/ocs/ui/views.py [hunk 4]
```diff
@@ -1745,21 +1790,12 @@
...
     ),
 }
 
-
-alerting = {
-    "alerts-tab-link": ("Alerts", By.LINK_TEXT),
-    "silences-tab-link": ("Silences", By.LINK_TEXT),
-    "alerting-rules-tab-link": ("Alerting rules", By.LINK_TEXT),
-    "runbook_link": ("//a[@class='co-external-link']", By.XPATH),
-    "alerting_rule_details_link": ("//a[normalize-space()='{}']", By.XPATH),
```
[the remark this answers is not in the document]
Comment (ebondare) [reply ocs_ci/ocs/ui/views.py]: fixed

File: ocs_ci/ocs/ui/views.py (hunk 2 above)
Comment (ebondare) [reply ocs_ci/ocs/ui/views.py]: fixed

File: ocs_ci/ocs/ui/views.py (hunk 3 above)
Comment (ebondare) [reply ocs_ci/ocs/ui/views.py]: fixed

**Review by petr-balogh — APPROVED** [review]

**Review by vavuthu — APPROVED** [review]

**Comment by DanielOsypenko** [comment]
can you pls rerun tests/functional/ui/test_alert_text.py::test_runbooks to doublecheck that no artifacts left.

**Comment by DanielOsypenko** [comment]
what test/tests this PR suppose to fix? can you pls add to description, so we can track and backport to 4.15 if needed

**Comment by ebondare** [comment]
> can you pls rerun tests/functional/ui/test_alert_text.py::test_runbooks to doublecheck that no artifacts left.

https://ocs4-jenkins-csb-odf-qe.apps.ocp-c1.prod.psi.redhat.com/job/qe-deploy-ocs-cluster/37362/parameters/ runbooks  test passed

**Comment by ebondare** [comment]
Test run with passing test_obc_creation_and_deletion https://ocs4-jenkins-csb-odf-qe.apps.ocp-c1.prod.psi.redhat.com/job/qe-deploy-ocs-cluster/37379/testReport/

**Review by DanielOsypenko — APPROVED** [review]

**Review by fbalak — APPROVED** [review]

**Comment by OdedViner** [comment]
please squash the commits in this pr

```
)$ git log 
commit be60a9df90d9542048c0743502fca4de3944a91a (HEAD -> master, tag: stable-ocs-4.16-202405160500, upstream/stable, upstream/master)
Author: Elena Bondarenko <40801241+ebondare@users.noreply.github.com>
Date:   Wed May 15 14:54:19 2024 +0200

    Adjust locators to patternfly5 (#9691)
    
    * Adjust locators to patternfly5
    
    Signed-off-by: Elena Bondarenko <ebondare@redhat.com>
    
    * Fix accidental deletion
    
    Signed-off-by: Elena Bondarenko <ebondare@redhat.com>
    
    * Fix tox
    
    Signed-off-by: Elena Bondarenko <ebondare@redhat.com>
    
    * Fix typo
    
    Signed-off-by: Elena Bondarenko <ebondare@redhat.com>
    
    * Fix locator for actions
    
    Signed-off-by: Elena Bondarenko <ebondare@redhat.com>
    
    * fix typo
    
    Signed-off-by: Elena Bondarenko <ebondare@redhat.com>
    
    * Adjust delete locator
    
    Signed-off-by: Elena Bondarenko <ebondare@redhat.com>
    
    * Fix obc deletion via UI
    
    Signed-off-by: Elena Bondarenko <ebondare@redhat.com>
    
    * Fix bucketclass search
    
    Signed-off-by: Elena Bondarenko <ebondare@redhat.com>
    
    * Fix actions locator
    
    Signed-off-by: Elena Bondarenko <ebondare@redhat.com>
    
    * Add another actions locator
    
    Signed-off-by: Elena Bondarenko <ebondare@redhat.com>
    
    * Fix wait for disabled button
    
    Signed-off-by: Elena Bondarenko <ebondare@redhat.com>
    
    ---------
    
    Signed-off-by: Elena Bondarenko <ebondare@redhat.com>

```

**Pull request merged**